fineract-provider/build.gradle

```diff
@@ -94,7 +94,7 @@ dependencyManagement {
         dependency 'com.lowagie:itext:2.1.7'
         dependency 'com.lowagie:itext-rtf:2.1.7'
         dependency 'org.mnode.ical4j:ical4j:1.0.7'
-        dependency 'com.googlecode.flyway:flyway-core:2.1.1'
+        dependency 'org.flywaydb:flyway-core:6.4.1'
```

Comment:
@xurror wait, waaaat - this work now?! Could I invite you to first raise a small separate PR just re. this, referencing (and closing?) FINERACT-810 ?